.gitlab-ci.yml

```diff
@@ -23,6 +23,7 @@ variables:
   BLUECONFIGS_BRANCH:
     value: $BLUECONFIGS_BRANCH
     description: 'Name of the blueconfigs branch to test against'
+  GIT_DEPTH: 100
```

Comment:
I hope we don't have to change this again... :)

Reply:
The original intent (per the first commit) was to apply that to the other jobs, which still had a default of 10. Since I had to fix it, sorry I applied my beloved 100 number to the whole thing